url.bs

```diff
@@ -36,9 +36,11 @@ Boilerplate: omit feedback-header, omit conformance
 
  <li><p>Standardize on the term URL. URI and IRI are just confusing. In
  practice a single algorithm is used for both so keeping them distinct is
- not helping anyone. URL also easily wins the
- <a href="http://www.googlefight.com/index.php?word1=url&amp;word2=uri">search result popularity contest</a>.
-
+ not helping anyone. 
+ An even if this a weak argument, URL also easily wins 
+ <a href="https://trends.google.com/trends/explore?q=url,uri">
+ search result popularity contest</a>
```

Comment:
I prefer keeping the old text. I don't think it's a weak argument per se. When having to chose between two terms using the one that's most popular is rather good practice as it requires the least amount of people to change habits.

And there's also no wrapping inside an inline element.